reactor-netty-http/src/main/java/reactor/netty/http/server/logging/error/ErrorLog.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2025 VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty.http.server.logging.error;
+
+import reactor.util.Logger;
+import reactor.util.Loggers;
+
+import java.util.Objects;
+
+/**
+ * Log the http error information into a Logger named {@code reactor.netty.http.server.ErrorLog} at ERROR level.
+ * <p>
+ * See {@link ErrorLogFactory} for convenience methods to create an access log factory to be passed to
+ * {@link reactor.netty.http.server.HttpServer#errorLog(boolean, ErrorLogFactory)} during server configuration.
+ *
+ * @author raccoonback
+ */
+public final class ErrorLog {
```

Comment:
Recently there was comment that `AccessLog` is final class which prevents the extensibility. May be we can consider the comment here https://github.com/reactor/reactor-netty/issues/781#issuecomment-2776094005 and make the `ErrorLog` interface?

Reply:
@violetagg 
Yes, that's a good idea. 
I'll consider using ErrorLog as an interface.